TimescaleDB 2.5 Release Notes (#517)
timescale/docs: timescaledb/overview/release-notes/index.md

behaviors to suit your needs.  
Some of these API updates are **breaking changes**.

### What's new in TimescaleDB 2.5:

> - Continuous aggregates for Distributed Hypertables

Reviewer: ```suggestion
- Continuous aggregates for distributed hypertables
```

> - Support for Postgres 14

Reviewer: ```suggestion
- Support for PostgreSQL 14
```

> - Experimental: Suport for timezones in 'time_bucket_ng()', including the 'origin' argument 

Reviewer: ```suggestion
- Experimental: Support for timezones in 'time_bucket_ng()', including the 'origin' argument 
```

> You can read more about this release on our [blogpost](TODO).

Reviewer: ```suggestion
You can read more about this release on our [blog post](TODO).
```

This release contains bug fixes since the 2.4.1 release. 

> We deem it high priority to upgrade since it includes a correction for 'compress_policy'.

Reviewer: ```suggestion
We highly recommend that you upgrade as soon as possible, because this update includes a correction for a 'compress_policy' bug.
```

Author: Just curious, Engineering usually ranks the release between high, medium and low.
How would these 3 cases be in this new phrasing?

Reviewer: I was trying to soften it from "next available opportunity". What about "when you are able to", or just "when you can"?

The experimental features in the 2.5 release are:
* APIs for chunk manipulation across data nodes in a distributed
hypertable setup. This includes the ability to add a data node and move
chunks to the new data node for cluster rebalancing.
* The `time_bucket_ng` function, a newer version of `time_bucket`. This function

> supports years, months, days, hours, minutes, and seconds and timezones.

Reviewer: ```suggestion
supports years, months, days, hours, minutes, seconds, and timezones.
```

Timescale is working hard on our next exciting features.
To make that possible, we require functionality that is available in Postgres 12 and above.
For this reason, we removed support for PostgreSQL 11 starting in the TimescaleDB 2.4 release. For TimescaleDB 2.4 and onwards, PostgreSQL 12 or 13 are required.

<highlight type="tip">

> TimescaleDB 2.5 is currently GA, and we encourage

Reviewer: ```suggestion
TimescaleDB 2.5 is now available, and we encourage
```
